recipe/meta.yaml

```diff
@@ -65,41 +65,41 @@ requirements:
     - posix                    # [win]
 
     # unknown
-    - {{native}}libjpeg-turbo  # [win]
-    - {{native}}libiconv       # [win]
-    - {{native}}gmp            # [win]
-    - {{native}}fftw           # [win]
-    - {{native}}mpfr           # [win]
-    - {{native}}libsndfile     # [win]
-    - {{native}}bwidget        # [win]
-    - {{native}}tktable        # [win]
+    - {{native}}libjpeg-turbo 1.4.*  # [win]
+    - {{native}}libiconv 1.14        # [win]
+    - {{native}}gmp 6.1.*            # [win]
+    - {{native}}fftw 3.3.*           # [win]
+    - {{native}}mpfr 3.1.*           # [win]
+    - {{native}}libsndfile 1.0.*     # [win]
+    - {{native}}bwidget 1.9.*        # [win]
+    - {{native}}tktable 2.10         # [win]
```

Comment:
This pinning will prevent using that version of R when I update the MSYS2 packages. This is far too rigid.